config/keys.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
just export without module.exports
like this 

const mongoUrl = `xxxxx`

export default {
    mongoUrl
}